packages/structure/src/outline/outline.ts

```diff
@@ -24,20 +26,25 @@ export function getOutline(project: RWProject): TreeItem2 {
         ),
         menu: {
           kind: 'withDoc',
-          doc: Command_open(
-            'https://redwoodjs.com/docs/app-configuration-redwood-toml'
-          ),
+          doc: Command_open(`${DOCS}/app-configuration-redwood-toml`),
         },
       } as TreeItem2,
       {
         label: 'open graphql playground',
         command: Command_open('http://localhost:8911/graphql'),
         iconPath: 'x-graphql',
+        menu: {
+          kind: 'withDoc',
+          doc: Command_open(
+            'https://www.apollographql.com/docs/apollo-server/testing/graphql-playground/'
+          ),
+        },
       } as TreeItem2,
       {
         label: 'open storybook',
         command: Command_cli('rw storybook --open'),
         iconPath: 'x-storybook',
+        // TODO: add link to rw storybook docs when available
```

Comment:
We have some cookbooks over here: https://redwoodjs.com/cookbook/mocking-graph-ql-in-storybook, whilst not strictly docs - I think it's better than nothing?

Reply:
sounds good to me